docs/src/references/child-devices.md

```diff
@@ -0,0 +1,124 @@
+   * Alarms, see [Raising an alarm to a child-device](../tutorials/raise-alarm.md#raising-alarms-from-child-devices)
+   * Configuration Management, see [Configuration files for child-devices](../references/c8y-configuration-management.md#configuration-files-for-child-devices)
+
+
+## User Journeys
+
+The user journeys below outline the behaviour of external-devices, thin-edge and the cloud during the provisioning phase.
+
+#### Personas used in user journeys
+* **device-operator**: the one who maintains/operates the device in the field in the shopfloor
+   
+#### User Journey 1: Connecting an external-device as new child-device, provisioning managed by device
+  - device-operator: connects new external-device to the thin-edge device by cable
+  - device-operator: powers up the external-device
+    - the external-device announces it-self to thin-edge with it's _child-id_ and capabilities (e.g. provided _configuration types_, _log types_, ...)
+      Details, see: [Child-Device provisioning](#1-child-device-provisioning) above
+    - thin-edge creates the child-device twin in the cloud, based on the _child-id_ reported by the external-evice
+
+      NOTE: Cloud's operation to create the child-device twin is idempotent, so trying to create a child-device twin again that already exist does not harm.
+    - thin-edge declares all capabilities reported by the external-device to the child-device twin (i.E. all _supported operations_ and corresponding types per operations, as _config-types_, _log-types_, ...)
+  - from that point the new external-device is ready in operation
+
+#### User Journey 2: Connecting an external-device as new child-device, provisioning managed by a cloud-site backend
+  - device-operator: creates the child-device twin in the cloud using some customer-specific backend; as _child-id_ the device-operator uses some UID that is known by the external-device (e.g. the external-device's serial number or network address)
+  - device-operator: declares all capabilities to the child-device twin (i.E. all _supported operations_ and corresponding types per operations, as _config-types_, _log-types_, ...)
+  - device-operator: connects new external-device to the thin-edge device by cable
+  - device-operator: powers up the external-device
+    - the external-device announces it-self to thin-edge with it's _child-id_
+    - thin-edge tries to created the child-device twin in the cloud, even though that twin was already created upfront
+
+      NOTE: Cloud's operation to create the child-device twin is idempotent, so trying to create a child-device twin that already exist does not harm.
+
+    - thin-edge retrieves all declared capabilities from the child-device twin (e.g. provided _configuration types_, _log types_, ...) and send as response to the external-device
+
+  - from that point the new external-device is ready in operation
+
```

Comment:
User journeys should be removed from that PR, but applied to PR "requirements and use-cases" (https://github.com/thin-edge/thin-edge.io/pull/1195).

Do not forget to consider user journeys pending comments:
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r944156805
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r947026933
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r944161015
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r944182229
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r947029394
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r944198937
  * https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r947033546
  
```suggestion

```